src/screens/Voucher/VoucherDetailScreen.tsx

```diff
@@ -164,6 +189,28 @@ export const VoucherDetailScreen = ({ navigation, route }: StackScreenProps<any>
           </Wrapper>
         </>
       )}
+
+      {!!voucherListItems?.length && (
+        <>
+          <SectionHeader title={texts.pointOfInterest.vouchersMore} />
```

Comment:
i have updated this to

<img width="372" alt="image" src="https://github.com/smart-village-solutions/smart-village-app-app/assets/1942953/2c7576fe-6bbe-4a4a-839f-7c00aa939049">